src/components/DpCheckboxGroup/DpCheckboxGroup.vue

```diff
@@ -22,6 +22,7 @@
 <script>
 import { CleanHtml } from '../../directives'
 import DpCheckbox from '../DpCheckbox/DpCheckbox'
+import Vue from "vue";
```

Comment:
we don't want to import vue here. Vue has to be accessible globally.

Reply:
Yes but I have this error message: `ReferenceError: "Vue is not defined".`

Reply:
I think what You actually want to do is something like https://github.com/demos-europe/demosplan-ui/pull/180/files

Reply:
or approve the PR and merge the changes into Your PR ;-)

Reply:
approved

Reply:
Merge done!

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/477/commits/92c9efaa1ffdb39447a262752008ec8c3378cba7